plugins/search-react/src/components/SearchBar/SearchBar.tsx

```diff
@@ -64,94 +63,97 @@ export type SearchBarBaseProps = Omit<InputBaseProps, 'onChange'> & {
  *
  * @public
  */
-export const SearchBarBase = ({
-  onChange,
-  onKeyDown,
-  onSubmit,
-  debounceTime = 200,
-  clearButton = true,
-  fullWidth = true,
-  value: defaultValue,
-  inputProps: defaultInputProps = {},
-  endAdornment: defaultEndAdornment,
-  ...props
-}: SearchBarBaseProps) => {
-  const configApi = useApi(configApiRef);
-  const [value, setValue] = useState<string>(defaultValue as string);
-  const hasSearchContext = useSearchContextCheck();
-
-  useEffect(() => {
-    setValue(prevValue =>
-      prevValue !== defaultValue ? (defaultValue as string) : prevValue,
+export const SearchBarBase = forwardRef<unknown, SearchBarBaseProps>(
+  (
+    {
+      onChange,
+      onKeyDown = () => {},
+      onClear = () => {},
+      onSubmit = () => {},
+      debounceTime = 200,
+      clearButton = true,
+      fullWidth = true,
+      value: defaultValue,
+      inputProps: defaultInputProps = {},
+      endAdornment: defaultEndAdornment,
+      ...props
+    },
+    ref,
+  ) => {
+    const configApi = useApi(configApiRef);
```

Comment:
can we destruct the components props here instead? I think that will improve the readability of the api reports a bit.